packages/errpy/errpy.0.0.9/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Errpy: An Error Recovering Python Parser implemented in Rust"
+description:
+  "This is a new project to provide an error-recovering Python parser, implemented in Rust based on tree-sitter. Our short-term goal is to use it in the pyre type checker, in order to provide better IDE features."
+maintainer: ["pyre@meta.com"]
+authors: ["pyre@meta.com"]
+license: "MIT"
+homepage: "https://github.com/facebook/errpy"
+bug-reports: "https://github.com/facebook/errpy/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.4"}
+  "ppx_deriving"
+  "conf-rust-2021" {build}
+  "ounit2" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/facebook/errpy.git"
+url {
+  src: "https://github.com/facebook/errpy/archive/refs/tags/0.0.9.tar.gz"
+  checksum: [
+    "md5=ec89b3c48081a43a3b9d8b4216fe2dad"
+    "sha512=a553863a0d80188e2dc52f5cc7b153fbe51f71f8cb991c0c7a524cbe3a000952148150de13daf90032a9723aeb991a666340627502b04ec0f7aeee295f0b86fd"
+  ]
+}
+post-messages: [
+  "ERRPY requires a rustc version of 1.65+ due to the tree-sitter dependency" {rustc:version < "1.65"}
```

Comment:
```suggestion
  "ERRPY requires a rustc version of 1.65+ due to the tree-sitter dependency" {failure}
```
This won't work, opam does not know rustc version. We need to use "failure" here to ensure it is printed each time the package fail